.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_LOCATION_KEY=pk.bf571c5497c491af2ef057dc7358c84f
+
+REACT_APP_WEATHER_KEY=6111091c47f743c7bc1410fabffbbb88
+
+REACT_APP_MOVIE_KEY=637d69539e64f605b5d28ed0a9519e57
+
+REACT_APP_YELP_KEY=OQAY3jiadm6MIpA1yFbiZWKHSq3nRKDVUzBux8hnmHm_P8NSqnWRTJjZdIUo9bNYhaPe3q8wjPp3193GPweFht7IBUVUGXkz_ATojIKDRkzx2ouCpeX6WgwO-1WBYXYx
```

Comment:
Hi there!

Make sure to get this .gitignored as soon as you can. We want to make sure this is hidden. Ensure that `.env` is added into `.gitignore`. You may need to grab TA to ensure that this is ignored properly since it is already committed. 👍